showcase/src/app/home/home.component.html

```diff
@@ -1,43 +1,100 @@
-<div>
-	<div class="mb3">
-		<mat-card>
+<h1 class="mat-display-2" translate>Stark documentation</h1>
+
+<div fxLayout="column" fxLayout.gt-xs="row wrap" fxLayoutAlign.gt-md="space-between center" fxLayoutAlign.md="space-around center"
+ fxLayoutAlign.lt-md="center center" fxLayoutGap.lt-lg="20px">
+	<a href="https://stark.nbb.be/api-docs/stark-core/latest">
+		<mat-card class="stark-card">
 			<mat-card-header>
 				<mat-card-title>
-					<h1 class="documentation-title">Stark documentation</h1>
+					<h4 translate>Stark Core - Latest</h4>
```

Comment:
For the titles of the cards of this Home page, could we have them centered horizontally?